etc/hub.api.md

```diff
@@ -909,13 +908,14 @@ export const Where: typeof Criterion;
 
 // @public
 export interface WithKeyInfoOptions extends WithUserAuthOptions {
-    // (undocumented)
     date?: Date;
+    debug?: boolean;
+    host?: string;

```

Comment:
Isn't the `host` and `debug` properties implied if it extends the `WithUserAuthOptions` type with those properties in it? Why the nested extensions if they are duplicated?

Reply:
it's all about the static docs generation. unfortunately, our generator doesn't pick up the implied properties.